etl/steps/data/garden/oecd/2025-07-03/affordable_housing.py

```diff
@@ -0,0 +1,188 @@
+"""Load a snapshot and create a meadow dataset."""
+
+import pandas as pd
+from owid.catalog import processing as pr
+
+from etl.data_helpers import geo
+from etl.helpers import PathFinder
+
+# Get paths and naming conventions for current step.
+paths = PathFinder(__file__)
+
+
+BREAK_STR = ".. Not available,  I break in series"
+
+
+def run() -> None:
+    # Load inputs.
+
+    snap_1_1 = paths.load_snapshot("affordable_housing_income.xlsx")
+    snap_1_2 = paths.load_snapshot("affordable_housing.xlsx")
+
+    # Load and clean tables from the snapshots.
+    tb_soc = snap_1_1.read_excel(
+        sheet_name="Figure HC1.1.2", usecols="J:AL", header=2
+    )  # housing costs share of consumption
```

Comment:
I would load all the tables here first for clarity, and then I would process them one by one